lib/std/os/bits/darwin.zig

```diff
@@ -1744,3 +1744,35 @@ pub const IOCPARM_MASK = 0x1fff;
 fn ior(inout: u32, group: usize, num: usize, len: usize) usize {
     return (inout | ((len & IOCPARM_MASK) << 16) | ((group) << 8) | (num));
 }
+
+// CPU families mapping
+pub const CPUFAMILY_UNKNOWN = 0;
```

Comment:
```suggestion
pub const CPUFAMILY = extern enum(u32) {
  UNKNOWN = 0,
  POWERPC_G3 = 0xcee41549,
  // ....
  _,
};
```

Reply:
Aha, I was not sure if we wanted to turn the system constants into enums where it made sense.
It does look much better tho, I'll change the defs.